crev-data/src/lib.rs

```diff
@@ -28,3 +26,69 @@ pub use crate::{
 
 #[cfg(test)]
 mod tests;
+
+type Result<T> = std::result::Result<T, Error>;
+
+#[derive(thiserror::Error, Debug)]
+pub enum Error {
```

Comment:
I always wonder if people split this into groups of some kind, or do they put all of the together.